resourcemanager/pool/spool/spool.go

```diff
@@ -79,8 +79,20 @@ func (p *Pool) Tune(size int32) {
 	p.mu.Lock()
 	defer p.mu.Unlock()
 	p.SetLastTuneTs(time.Now())
+	old := p.capacity
 	p.capacity = size
 	p.concurrencyMetrics.Set(float64(size))
+	if old < size {
```

Comment:
If `old == size`, then do nothing?

Reply:
Yes， I refactor it. Now if old == size, return directly.